integration-tests/tests/billing-projects.js

```diff
@@ -1,12 +1,15 @@
 // This test is owned by the Workspaces Team.
 const _ = require('lodash/fp')
-const { assertTextNotFound, click, clickable, findText, noSpinnersAfter, select, signIntoTerra } = require('../utils/integration-utils')
+const { assertTextNotFound, click, clickable, findText, noSpinnersAfter, select, signIntoTerra, waitForNoSpinners } = require('../utils/integration-utils')
 const { withUserToken } = require('../utils/terra-sa-utils')
 
 
 const billingProjectsPage = (testPage, testUrl) => {
   return {
-    visit: async () => await testPage.goto(`${testUrl}/#billing`),
+    visit: async () => {
+      await testPage.goto(`${testUrl}/#billing`)
```

Comment:
I tried this but put it back.. I don't think `noSpinnersAfter` works when the action is `page.goto(…)` because it tries to verify the spinner appears and disappears on a single page.

Reply:
OK, thanks!

Reply:
It's actually interesting because there is a similar conversation going on here-- https://github.com/DataBiosphere/terra-ui/pull/3021/files#r870656864. My thought with `noSpinnersAfter` was to avoid this sort of problem, but I understand why it doesn't work. I can keep an eye on the test though and try to change to the promise approach Alex is suggesting if we see failures.